src/resmom/prolog.c

```diff
@@ -382,7 +396,8 @@ int   pe_io_type;
 		run_exit = 254;
 		(void)pelog_err(pjob, pelog, run_exit, "nonzero p/e exit status");
 		return run_exit;
-	}
+	} else if (pelog_handle == NULL)
```

Comment:
why do you want to continue from here instead of exit?

Reply:
it could crash the program because we are using handle at line 452 or am i missing something here ?

Reply:
> it could crash the program because we are using handle at line 452 or am i missing something here ?

Yes you are right. I will put this check in the if statement above on line 395.